src/lib.rs

```diff
@@ -891,5 +897,19 @@ mod tests {
         let mime2 = Mime::from_str(r#"text/x-custom; aBc=a"#).unwrap();
         assert_eq!(mime1, mime2);
     }
+
+    #[cfg(feature = "serde")]
+    #[test]
+    fn serde_tests() {
```

Comment:
Probably should be called `test_serde`, as this seems to be naming convention in this crate.